runners/direct-java/src/main/java/org/apache/beam/runners/direct/WatermarkManager.java

```diff
@@ -597,20 +597,29 @@ public synchronized Instant getEarliestTimerTimestamp() {
       Instant earliest = THE_END_OF_TIME.get();
       for (NavigableSet<TimerData> timers : processingTimers.values()) {
```

Comment:
What about the other getEarliestTimerTimestamp function? Does that need to be updated?

Reply:
@reuvenlax  updated